resources/views/components/post-item.blade.php

```diff
@@ -34,10 +34,6 @@
     </div>
 
     <div class="mt-2">
-        @if ($post->formatting == 'rich')
-        {!! $post->content !!}
-        @else
-        {!! nl2br(e($post->content)) !!}
-        @endif
+        {!! $post->isRichlyFormatted ? $post->content : nl2br(e($post->content)) !!}
```

Comment:
What about doing `getContentAttribute()` and adding this to it?

Reply:
I don't love overriding an actual column name with an attribute value, how do you feel about `getFormattedContentAttribute()`?